salt/journal-cms/init.sls

```diff
@@ -319,6 +319,14 @@ site-cache-rebuild-again:
         - require:
             - site-configuration-import
 
+site-permissions-rebuild:
+    cmd.run: ../vendor/bin/drush php-eval "node_access_rebuild();"
```

Comment:
I'm not sure about the notation inline with `cmd.run`, usually it is specified as a separate `name:` field before `cwd:` and `user:`